packages/openapi-v3/src/__tests__/unit/filter-schema.unit.ts

```diff
@@ -21,13 +21,54 @@ describe('filterSchema', () => {
     const schema = getFilterSchemaFor(MyUserModel);
     expect(MyUserModel.definition.name).to.eql('my-user-model');
     expect(schema).to.eql({
+      title: 'my-user-model.Filter',
       properties: {
         where: {
           type: 'object',
+          title: 'my-user-model.WhereFilter',
           additionalProperties: true,
         },
         fields: {
           type: 'object',
+          title: 'my-user-model.Fields',
+          properties: {
+            id: {type: 'boolean'},
+            age: {type: 'boolean'},
+          },
+          additionalProperties: false,
+        },
+        offset: {type: 'integer', minimum: 0},
+        limit: {type: 'integer', minimum: 1, example: 100},
+        skip: {type: 'integer', minimum: 0},
+        order: {type: 'array', items: {type: 'string'}},
+      },
+      additionalProperties: false,
+    });
+  });
+
+  @model({
+    name: 'CustomUserModel',
+  })
+  class CustomUserModel extends Entity {
+    @property() id: string;
+
+    @property() age: number;
+  }
+
+  it('generate filter schema with custom name', () => {
```

Comment:
Nitpick:

```suggestion
  it('generates filter schema with custom name', () => {
```

As in an English sentence: _it generates filter schema etc._